### ISSUE TITLE
bug: [Cortex ps  fetching the updated models status]

### ISSUE THREAD

**Issue by TC117** [janhq/cortex.cpp#1641]
### Cortex version

v1.0.1

### Describe the issue and expected behaviour

 cortex ps isn't fetching the updated status because I already deleted the model tinyllama:main but it still show up

### Steps to Reproduce

1. cortex run tinyllama:main
2. cortex models delete tinyllama:main
3. cortex ps

### Screenshots / Logs

![image](https://github.com/user-attachments/assets/4964d4a5-555e-4d7d-aa6d-80483a1cc7a7)

### What is your OS?

- [X] Windows
- [ ] Mac Silicon
- [ ] Mac Intel
- [ ] Linux / Ubuntu

### What engine are you running?

- [X] cortex.llamacpp (default)
- [ ] cortex.tensorrt-llm (Nvidia GPUs)
- [ ] cortex.onnx (NPUs, DirectML)

### Hardware Specs eg OS version, GPU

Win11

**Comment by namchuai** [comment]
<img width="978" alt="Screenshot 2024-11-07 at 13 06 39" src="https://github.com/user-attachments/assets/7046d3c5-032a-42da-b2e6-5d9258e44f1f">
I think this issue is fixed in `dev`. Should be available to test on latest nightly version.

**Comment by gabrielle-ong** [comment]
Fixed for me in Mac nightly v230. 
@TC117 will let you test on Windows and close this ticket if resolved 
@namchuai thanks for fixing! Can you share the PR that fixed this?

![Image](https://github.com/user-attachments/assets/6c317e7b-5526-4e51-8f7f-d5fb79dbf7e1)

**Comment by TC117** [comment]
```
PS C:\WINDOWS\system32> cortex-nightly.exe ps
+----------------+-----------+----------+-----------+------------+
| Model          | Engine    | RAM      | VRAM      | Up time    |
+----------------+-----------+----------+-----------+------------+
| tinyllama:main | llama-cpp | 35.16 MB | 601.02 MB | 13 seconds |
+----------------+-----------+----------+-----------+------------+

New llama.cpp version available: v0.1.37
To update, run: cortex-nightly.exe engines update llama-cpp
PS C:\WINDOWS\system32>
PS C:\WINDOWS\system32> cortex-nightly.exe delete tinyllama:main
The following arguments were not expected: tinyllama:main delete
Run with --help for more information.
PS C:\WINDOWS\system32> cortex-nightly.exe models delete tinyllama:main
Model tinyllama:main deleted successfully

New llama.cpp version available: v0.1.37
To update, run: cortex-nightly.exe engines update llama-cpp
PS C:\WINDOWS\system32> cortex-nightly.exe ps
No models loaded!

New llama.cpp version available: v0.1.37
To update, run: cortex-nightly.exe engines update llama-cpp
PS C:\WINDOWS\system32>
```
same with engines
```
PS C:\WINDOWS\system32> cortex-nightly.exe engines uninstall llama-cpp
Engine llama-cpp uninstalled successfully!

New llama.cpp version available: v0.1.37
To update, run: cortex-nightly.exe engines update llama-cpp
PS C:\WINDOWS\system32> cortex-nightly.exe ps
No models loaded!

New llama.cpp version available: v0.1.37
To update, run: cortex-nightly.exe engines update llama-cpp
PS C:\WINDOWS\system32> cortex-nightly.exe -v
v1.0.2-230

New llama.cpp version available: v0.1.37
To update, run: cortex-nightly.exe engines update llama-cpp
PS C:\WINDOWS\system32>
``` 

Thank you, checked on Win11 nightly 230, confirm issue is fixed
